proposals/nnnn-video-streaming-state.md

```diff
@@ -0,0 +1,165 @@
+# Enhancing onHMIStatus with a New Parameter for Video Streaming State
+
+* Proposal: [SDL-NNNN](nnnn-video-streaming-state.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC]
+
+## Introduction
+
+This proposal adds a new parameter `videoStreamingState` to `onHMIStatus` notification so that the SDL can notify an app stop video streaming. This helps SDL handle the use cases of the coexistence of a navigation app and a projection app or multiple navigation/projection apps.
```

Comment:
I think the first sentence should be updated to "...so that SDL Core can notify an app to stop video streaming."  The second sentence should also be updated to "This helps SDL Core..."

Reply:
update as suggested.